Emby.Server.Implementations/EntryPoints/UserDataChangeNotifier.cs

```diff
@@ -97,10 +99,10 @@ private void UpdateTimerCallback(object state)
 
                 var task = SendNotifications(changes, CancellationToken.None);
 
-                if (UpdateTimer != null)
+                if (_updateTimer != null)
                 {
-                    UpdateTimer.Dispose();
-                    UpdateTimer = null;
+                    _updateTimer.Dispose();
+                    _updateTimer = null;
```

Comment:
```c#
_updateTimer?.Dispose();
_updateTimer = null;
```
?

Reply:
Why? We check the line before if it's not null.